src/store/auth/operations/loginOperation.js

```diff
@@ -0,0 +1,40 @@
+import { createLogic } from 'redux-logic'
+// eslint-disable-next-line import/no-unresolved
+import { LOGIN_SUBMIT } from '../types'
+import { loginLoading, loginError, loginSuccess } from '../actions'
+
+export const loginOperation = createLogic({
+  type: LOGIN_SUBMIT,
+  latest: true,
+  async process({ action, axios }, dispatch, done) {
+    dispatch(loginLoading(true))
+    dispatch(loginError(null))
```

Comment:
you can create action `loginRequest` which will set `error` to `null` and `loading` to `true` instead

Reply:
do you plan to fix comment above?